README.md

```diff
@@ -21,6 +21,10 @@ endpoints from before Matrix 1.1, for example.
 
 # Quickstart
 
+> [!IMPORTANT]
```

Comment:
for the reviewer - I think this our first usage of the blockquote styles provided by github. See https://github.com/orgs/community/discussions/16925 for details.